charts/cryostat/templates/db_deployment.yaml

```
@@ -85,12 +85,12 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       volumes:
-      {{- if ((.Values.pvc).enabled) }}
+      {{- if ((.Values.db.resources.pvc).enabled) }}
       - name: {{ .Chart.Name }}-db
         persistentVolumeClaim:
           claimName: {{ .Release.Name }}-db
       {{- end }}
-      {{- if not ((.Values.pvc).enabled) }}
+      {{- if not ((.Values.db.resources.pvc).enabled) }}
```

```suggestion
      {{- else }}
```

I guess we can use `else` here?